ext/cdi/jersey-cdi-inject/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>project</artifactId>
+        <groupId>org.glassfish.jersey.ext.cdi</groupId>
+        <version>2.34-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jersey-cdi-inject</artifactId>
```

Comment:
It may be nice to get the word `context` in the artifact ID. CDI is already part of the group ID.

Reply:
the word `cdi` corresponds to other modules, all of them have cdi1x in the artifact id. The 1x is no longer used though, we use 2x (and even 3x in Jersey 3), so I left the 1x behind. I thought about `jersey-cdi-context-inject`, but I am not sure it is any better. Do you have any particular name in mind?

Reply:
Upon further consideration, `jersey-cdi-jaxrs-inject` seems to be a more descriptive name.

Reply:
@jansupol as "jaxrs" is the old Java EE name, what about considering the new Jakarta name? E.g. "Jakarta Rest" or just "rest" here?

Reply:
@arjantijms Thank you to chime in. The naming conventions seem to be the hardest on the overall work. I agree that "jaxrs" is not the best we could use in Jakarta EE. For Jersey 2.x it is far more descriptive than any other name I heard. For 3.x, changing to "rest" sounds a bit ambiguous. One can create REST endpoints with JAX-WS, for instance. "jakarta-rest" on the other hand sounds too long. Ideally, the name should be the same for 2.x and 3.x. `jersey-cdi-rs-inject` perhaps? I am open to any suggestions.

Reply:
`jersey-cdi-rs-inject` it is.

Reply:
JAX-RS is neither old nor new nor forbidden or obsolete. Most people understand what JAX-RS means and clearly *the technology* itself is mostly referred to still as JAX-RS. As I explained several times, it is officially still allowed and legal to keep using the word JAX-RS anywhere. One *the specification title* (hence, reference to *the specification*, not *the technology*) must be named "Jakarta RESTful Web Services". So it would have been absolutely ok that you keep "jaxrs" in the name if you like to do that. Just to make things clear, as there is much FUD about this. :-)

Reply:
This is likely for a discussion with a larger audience. The official Jakarta documents [do not like JAX](https://jakarta.ee/legal/acronym_guidelines/) prefix. I know you keep saying you have the approval, but no one has seen it. No matter the legal stuff, the javax based vs jakarta based Jersey differences are better explained to the customers when each has a different name, JAX-RS for Java EE 8, Jakarta REST for Jakarta EE 9.

Reply:
Don't get me wrong - I am totally OK with your decision. I just wanted to make clear that there is absoutely **no** legal reason that *forces* you to get rid of the word "JAX-RS". The document you correctly refer to just says "SHOULD" not "MUST", and it clearly asks for the change *only* for documents etc, but *not* for source code and other technical stuff:
> For **names of Jakarta EE projects**, **specifications**, **test suites**, and other **related materials**, the full **project name/title** and any **scope statement** SHOULD not incorporate an identified acronym.

The document **even allows** the use of the "old" acronyms explicitly for technical stuff:
> A **URL**, **repository name**, **package name**, **class name**, or **method name** MAY include the acronym to the right of a string identifying Eclipse or Jakarta as the origin.

It is completely up to the Jersey committers to decide whether they still use the acronym JAX-RS or not. No instance at the Eclipse Foundation or at Oracle explicitly *forbids* it. The document you refer to makes this 100% clear, so I am not sure why you say, "no one has seen it": It is written THERE. THAT document IS the final answer. So there is no more FUD. :-)